.prettierrc

```diff
@@ -3,5 +3,5 @@
   "semi": true,
   "singleQuote": true,
   "printWidth": 120,
-  "endOfLine":"auto" 
+  "endOfLine": "auto"
 }
```

Comment:
I've got problems with configuration, because stylelint removes empty lines in scss files onSave. Moreover files like .prettierrc, .eslintrc, .stylelintrc.json are not fixed by any lintier and automatically removes last empty lines onSave too. I tried to find a solution, but right now nothing works :( 
.ts/tsx files are formatting correctly, .scss files are formating incorrectly onSave, but correctly after --fix script